js/README.md

```diff
@@ -202,11 +197,33 @@ The JS community is a diverse group with a varied list of target environments an
 
 If you think we missed a compilation target and it's a blocker for adoption, please open an issue.
 
+# Community
+
+Full list of broader Apache Arrow [committers](https://arrow.apache.org/committers/) and [projects & organizations](https://github.com/apache/arrow/blob/master/site/powered_by.md).
+
+## Committers
+
+* Brian Hulette, CCRI
+* Paul Taylor, Graphistry, Inc.
+
+## Open Source Projects
+
+* [Apache Arrow](https://arrow.apache.org) -- Parent project for Powering Columnar In-Memory Analytics, including affiliated open source projects
+* [rxjs-mapd](https://github.com/graphistry/rxjs-mapd) -- A MapD Core node-driver that returns query results as Arrow columns
+
+
+
+## Companies & Organizations
+
+* [CCRi](http://www.ccri.com/) -- Commonwealth Computer Research Inc, or CCRi, is a Central Virginia based data science and software engineering company
+* [GoAI](http://gpuopenanalytics.com/) -- Open GPU-Accelerated Analytics Initiative for Arrow-powered analytics across GPU tools and vendors
+* [Graphistry, Inc.](https://www.graphistry.com/) - Supercharged Visual Investigation Platform used by teams for security, anti-fraud, and related investigations. The Graphistry team uses Arrow in its NodeJS GPU backend and client libraries, and is an early contributing member to GOAI and Arrow\[JS\] focused on bringing these technologies to the enterprise.
```

Comment:
Can we remove this section altogether and include a link in the readme to the "Powered By" page on the website (we should confirm, but I think it will be arrow.apache.org/powered-by.html)?

Reply:
I intentionally put social proof about the JS bindings here as it is new and relevant information for adoption decisions. The page is short enough that there seems more value putting them here than elsewhere -- once it's larger, I'd agree.